packages/cashier/src/types/shared/websocket.types.ts

```diff
@@ -29,6 +30,13 @@ type TWebSocketCall = {
 
 export type TWebSocket = {
     authorized: TWebSocketCall;
+    cryptoConfig: () => { crypto_config: CryptoConfig };
+    cryptoWithdraw: (args: {
+        address: string;
+        amount: number;
+        verification_code: string;
+        dry_run?: number;
+    }) => Promise<any>;
```

Comment:
```suggestion
    cryptoWithdraw: (
        args: Omit<CashierInformationRequest, 'cashier' | 'provider' | 'type'>
    ) => Promise<CashierInformationResponse>;
```

Reply:
Does it need to Omit these three properties? they are optional so they don't produce any ts error, however, maybe these properties will be used somewhere else.

Reply:
@hamid-deriv Yes we should omit those since they are hardcoded inside `cryptoWithdraw` and can’t be changed.